sql/config_test.go

```diff
@@ -92,7 +92,7 @@ func TestGetZoneConfig(t *testing.T) {
 	defer srv.Stopper().Stop()
 	s := srv.(*server.TestServer)
 
-	expectedCounter := uint32(keys.MaxReservedDescID + 1)
+	expectedCounter := uint32(keys.MaxReservedDescID)
```

Comment:
It doesn't. What fails is the pattern of post-incrementing, which results in the final increment being ineffectual.